sky/skylet/providers/kubernetes/config.py

```diff
@@ -128,11 +128,16 @@ def get_resource(container_resources: Dict[str, Any],
     # float('inf') means there's no limit set
     res_count = request if limit == float('inf') else limit
     # Convert to int since Ray autoscaler expects int.
-    # Cap the minimum resource to 1 because if resource count is set to 0,
-    # (e.g., when request=0.5), ray will not be able to schedule any tasks.
     # We also round up the resource count to the nearest integer to provide the
     # user at least the amount of resource they requested.
-    return max(1, math.ceil(res_count))
+    rounded_count = math.ceil(res_count)
+    if resource_name == 'cpu':
+        # For CPU, we set minimum count to 1 because if CPU count is set to 0,
+        # (e.g., when request=0.5), ray will not be able to schedule any tasks.
```

Comment:
From the comment, `request=0.5`, what is the `request` referring to? If it's referring to the `request` variable used in this function, `rounded_count`(CPU count) should be set to 1, not 0 when `request=0.5`. If it's not the variable, I think we should elaborate a bit more in the comment to show that it's not referring to the variable :)

Reply:
Good catch - clarified that it's referring to when skypilot task requests something like `--cpus 0.5`.